### PR TITLE
Update page content

### PR DESCRIPTION
**Pull request by israr-ulhaq**
Updating and fixing the page content, requirements from the design team.
Screenshot 2024-06-06 at 13 14 15

### CODE REVIEW

**Review by samuelhwilliams — APPROVED** [review]

File: app/templates/main/request-received.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 class="govuk-heading-m govuk-!-margin-top-7">
             We will email a link to <span class="govuk-!-font-weight-bold">{{ context["user_email"] }}</span>.
         </p>
         <p class="govuk-body">
-            This may take anything up to 5 minutes to come through to your inbox.</p>
+            This may take up to 5 minutes to come through to your inbox.</p>
```
Comment (Kateharries) [inline app/templates/main/request-received.html:18]: This may take up to 5 minutes to be delivered to your inbox.

**Review by Kateharries — CHANGES_REQUESTED** [review]
@israr-ulhaq I've left a small content adjustment on line 18.

File: app/templates/main/request-received.html (hunk 1 above)
Comment (israr-ulhaq) [reply app/templates/main/request-received.html:18]: Done!

**Review by Kateharries — APPROVED** [review]
looks great

**Pull request merged**